src/Nethermind/Nethermind.Blockchain/SpecificBlockReadOnlyStateProvider.cs

```diff
@@ -22,7 +22,7 @@ public SpecificBlockReadOnlyStateProvider(IStateReader stateReader, Hash256? sta
 
         public virtual Hash256 StateRoot { get; }
 
-        public AccountStruct GetAccount(Address address) => _stateReader.GetAccount(StateRoot, address) ?? default;
+        public AccountStruct GetAccount(Address address) => _stateReader.GetAccount(StateRoot, address) ?? Account.TotallyEmpty.ToStruct();
```

Comment:
Might be worth storing struct in readonly static rather than going via construstor in `.ToStruct`

Reply:
Lets add AccountStruct.TotallyEmpty